sdk/test-utils/perfstress/src/program.ts

```diff
@@ -204,13 +220,15 @@ export class PerfStressProgram {
       `\n=== ${title} mode, iteration ${iterationIndex}. Logs every ${millisecondsToLog /
         1000}s ===`
     );
-    console.log(`Since Last Log\t\tTotal`);
-    let lastInIteration = 0;
+    console.log(`Current\t\tTotal\t\tAverage`);
```

Comment:
```suggestion
    console.log(`Total\t\tCurrent\t\tAverage`);
```

Reply:
This would need to be changed in all 4 languages, and I don't think the alternative is better enough to be worth it.

Reply:
No, I meant.. the output has total first and then the current, the title says the opposite.
`console.log(`${totalCompleted}\t\t${currentCompleted}\t\t${averageCompleted.toFixed(2)}`);`

We could change the `console.log` instead.

Reply:
Got it.  The title is correct, the output needs to be fixed.  Good catch.